v1/gitlab/test-binary-fields-disable_overriding_approvers_per_merge_request.rego

```diff
@@ -0,0 +1,59 @@
+package verify
+
+import future.keywords.in
+
+default allow := false
+default violations := []
+
+default disable_overriding_approvers_per_merge_request := false
+
+disable_overriding_approvers_per_merge_request := input.config.args.disable_overriding_approvers_per_merge_request 
+
+verify = v {
+    v := {
+        "allow": allow,
+        "violation": {
+            "type": "Verify that the binary field: 'disable_overriding_approvers_per_merge_request' is set correctly",
+            "details": violations,
+        },
+        "summary": [{
+            "allow": allow,
+            "reason": reason,
+            "violations": count(violations),
+        }],
+    }
+}
+
+allow {
+    count(violations) == 0
+}
+
+reason = v {
+    allow
+    v := "All binary fields: 'disable_overriding_approvers_per_merge_request' are properly set "
+}
+
+reason = v {
+    not allow
+    v := "At least one of the binary fields: 'disable_overriding_approvers_per_merge_request' are not properly set"
+}
+
+
+
+# Violation has been switched here as a list instead of a set
+violations := [r |
+
+    project := object.remove(input.evidence.predicate.content, {"metadata"})[_]
+
+    # r := { "s": project.project.result_object.approval_settings.disable_overriding_approvers_per_merge_request}
+    disable_overriding_approvers_per_merge_request != project.project.result_object.approval_settings.disable_overriding_approvers_per_merge_request
+
+    r := {
+        "scribe_type": project.project.scribe_type,
+        "name": project.project.name,
+        "id": project.project.id,
+        "disable_overriding_approvers_per_merge_request": project.project.result_object.approval_settings.disable_overriding_approvers_per_merge_request,
+        "desired_disable_overriding_approvers_per_merge_request": disable_overriding_approvers_per_merge_request,
+    }
+] 
```

Comment:
Since you have only one check here and don't iterate through any objects, better move this to `allow` field to have a better focus and leave `violations` for additional details. Take a look at `v1/images/fresh-image.rego`, it's a good example of such check.